### PR TITLE
triton/apps/python-conda: New page with information on using conda

### PR DESCRIPTION
**Pull request by simo-tuomisto**
This PR creates a new page: Python environments with Conda.

Contents include:
- Adding a new page on conda environments.
- Sharing info-box on conda activate vs source activate between the new page and the old python page.
- Adding new example environments for tensorflow, pytorch and generic cuda environment.

### CODE REVIEW

File: triton/apps/python-conda.rst [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+
+Below we have a simple ``environment.yml``:
+
+.. code-block:: yaml
+
+  name: conda-example
+  channels:
+    - conda-forge
+  dependencies:
+    - blas * mkl
```
Comment (rkdarst) [inline triton/apps/python-conda.rst:54]: blas seems like something beyond what most peolpe would use.  what about an example of pinning a ML library to a specific version as the starting example?

**Review by rkdarst — APPROVED** [review]
Looks good, one small comment but of course we can keep improving...

File: triton/apps/python-conda.rst (hunk 1 above)
Comment (simo-tuomisto) [reply triton/apps/python-conda.rst]: @tpfau had the same comment. Fixed it in the new commit. MKL & BLAS is mentioned later in the complex environment section.

**Comment by simo-tuomisto** [comment]
This should be good to go.

**Review by tpfau — APPROVED** [review]
Looks Good

**Pull request merged**